src/components/shared/SearchInput.js

```diff
@@ -72,10 +72,10 @@ class SearchInput extends Component {
 
   renderSvg() {
     if (this.shouldShowErrorEmoji()) {
-      return <Svg name="sad-face" />;
+      return <Svg name="sad-face" role="presentation" />;
```

Comment:
If this is the only way the error is conveyed, it should not be presentation. Instead it should have a role of status https://www.w3.org/TR/wai-aria/roles#status or something similar. It should also have a textual alternative (e.g. title attribute or aria-label).

Reply:
The bug stated that the role should be "presentation", but instead it should be "status". Correct?